tests/test_legal.py

```diff
@@ -192,3 +193,66 @@ def test_query_dsl_with_ao_category_filter(self, es_search):
         es_search.assert_called_with(body=expected_query,
                                      index=mock.ANY,
                                      doc_type=mock.ANY)
+
+    @patch.object(es, 'search')
+    def test_query_dsl_with_mur_filters(self, es_search):
+        response = self.app.get('/v1/legal/search/', query_string={
+                                'q': 'embezzle',
+                                'type': 'murs',
+                                'mur_min_open_date': '2017-01-12'})
```

Comment:
I thought we added more clauses like `mur_max_open_date`, `mur_min_close_date`, etc.

Reply:
We did! I remember I couldn't push because of FEC wifi, and now I can't find the changes. I guess it's possible I overwrote them somehow?